addons/mail/static/src/owl/components/activity/activity.scss

```
@@ -0,0 +1,174 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_Activity {
+    display: flex;
+    flex: 0 0 auto;
+    padding: 5px;
+}
+
+.o_Activity_detailsUserAvatar {
+    margin-right: 5px;
+    object-fit: cover;
+    height: 18px;
+    width: 18px;
+}
+
+.o_Activity_icon {
+    position: absolute;
+    top: auto;
+    left: auto;
+    bottom: -5px;
+    right: -5px;
+    width: 25px;
+    height: 25px;
+    object-fit: cover;
+    padding: 6px 5px;
+    text-align: center;
+    line-height: 1.2;
+    border-width: 2px;
+}
+
+.o_Activity_sidebar {
+    display: flex;
+    flex: 0 0 36px;
+    margin-right: 10px;
+    justify-content: center;
+}
+
+.o_Activity_toolButton {
+    padding-top: 0;
+}
+
+.o_Activity_tools {
+    display: flex;
+}
+
+.o_Activity_user {
+    height: 36px;
+    margin-left: 3px;
+    margin-right: 3px;
+    position: relative;
+    width: 36px;
+}
+
+.o_Activity_userAvatar {
+    height: 100%;
```

for size with %, there is the $sizes variable.
```suggestion
    height: map-get($sizes, 100);
```